airflow/providers/amazon/aws/sensors/redshift_cluster.py

```diff
@@ -58,3 +58,19 @@ def get_hook(self) -> RedshiftHook:
 
         self.hook = RedshiftHook(aws_conn_id=self.aws_conn_id)
         return self.hook
+
+
+class AwsRedshiftClusterSensor(RedshiftClusterSensor):
```

Comment:
We dont need deprecation here.
This is a new class added recently in https://github.com/apache/airflow/pull/20276/files
This was never released to users so we can avoid this deprecation.

We should redirect from
https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/sensors/redshift.py

directly to the `RedshiftClusterSensor`